### ISSUE TITLE
Contact sensor "not currently supported" for triggering Alexa routine

### ISSUE THREAD

**Issue by surfahboy21** [ecoen66/homebridge-dummy-contact#3]
Hi there. Used your plugin to create a dummy switch and contact sensor. Then used NorthernMan54's homebridge-alexa plugin to expose both to Alexa. Setup worked well and both showed up in my list of devices in the Alexa app. Controlling the switch from the Home app also properly affected both the switch and the sensor states in the Alexa app. So then I proceeded to try and create a routine based in input from the sensor...and unfortunately I got an error message saying, "This device is not currently supported."

Any ideas what might be happening here? Thanks in advance for your help and assistance.

**Comment by NorthernMan54** [comment]
@surfahboy21 Pls share you config.   Do you have 'contact': true in your config ?

**Comment by surfahboy21** [comment]
Hi @NorthernMan54. Yes, the "contact" setting was "true" on the original dummy switch+sensor I set up / was having issues with. But today I logged into my HB admin panel and decided to try and delete that switch+sensor and create a new one to see if that would somehow solve things. Not sure why, but it worked. So everything is functioning as it should now. Appreciate your reaching out / being willing to investigate.
